mllib/src/test/scala/org/apache/spark/ml/tree/impl/RandomForestSuite.scala

```diff
@@ -176,6 +203,18 @@ class RandomForestSuite extends SparkFunSuite with MLlibTestSparkContext {
     assert(tree.rootNode.impurity === -1.0)
```

Comment:
oh, I see.  I've re-added back the map.  I also modified the second test with no categorical features to use regression instead of classification.